nss-day-cohort-46/holiday-road-unnamed-riddle
# Ap show itenerary

# Description

Currently this code displays all saved itineraries to the dom will need to change the call function in main.js and place it in the save Ititnerary button js file to show itineraries on save and refresh per saved itinerary. 
This will also need css to match the rest of the website.!!!
Fixes # (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions


Make sure you have itineraries api populated and then should display them to the dom.
git fetch --all
git checkout ap-ShowItenerary


# Checklist:
